x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_logic.ts

```diff
@@ -62,6 +62,61 @@ export const AnalyticsLogic = kea<MakeLogicType<AnalyticsValues, AnalyticsAction
         onQueryDataLoad: (_, { allTags }) => allTags,
       },
     ],
+    totalQueries: [
+      0,
+      {
+        onAnalyticsDataLoad: (_, { totalQueries }) => totalQueries,
+      },
+    ],
+    totalQueriesNoResults: [
+      0,
+      {
+        onAnalyticsDataLoad: (_, { totalQueriesNoResults }) => totalQueriesNoResults,
+      },
+    ],
+    totalClicks: [
+      0,
+      {
+        onAnalyticsDataLoad: (_, { totalClicks }) => totalClicks,
+      },
+    ],
+    queriesPerDay: [
+      [],
+      {
+        onAnalyticsDataLoad: (_, { queriesPerDay }) => queriesPerDay,
+      },
+    ],
+    queriesNoResultsPerDay: [
+      [],
+      {
+        onAnalyticsDataLoad: (_, { queriesNoResultsPerDay }) => queriesNoResultsPerDay,
+      },
+    ],
+    clicksPerDay: [
+      [],
+      {
+        onAnalyticsDataLoad: (_, { clicksPerDay }) => clicksPerDay,
+      },
+    ],
+    totalQueriesForQuery: [
+      0,
+      {
+        onQueryDataLoad: (_, { totalQueriesForQuery }) => totalQueriesForQuery,
+      },
+    ],
+    queriesPerDayForQuery: [
+      [],
+      {
+        onQueryDataLoad: (_, { queriesPerDayForQuery }) => queriesPerDayForQuery,
+      },
+    ],
+    startDate: [
+      '',
+      {
+        onAnalyticsDataLoad: (_, { startDate }) => startDate,
+        onQueryDataLoad: (_, { startDate }) => startDate,
+      },
+    ],
```

Comment:
I totally forgot charts need a `startDate`, so I re-added / re-stored this var. We shouldn't have to worry about it falling out of sync with our client-side `startDate` however since we now always send a `startDate` query param to the API (thanks Jason for that suggestion!)